pypit/armlsd.py

```diff
@@ -51,16 +51,17 @@ def ARMLSD(fitsdict, reuseMaster=False, reloadMaster=True):
     status = 0
 
     # Create a list of science exposure classes
-    sciexp = armbase.SetupScience(fitsdict)
-    numsci = len(sciexp)
+    sciexp, setup_dict = armbase.SetupScience(fitsdict)
+    if sciexp in ['setup', 'calcheck']:
+        return status
```

Comment:
status = 0 is for successful reduction. It might be a good idea to change the status value within the if statement, and return a different status number. See line ~233 of `pypit.py`, which will also need to be updated with the appropriate status number.